platforms-serverless/netlify-github/finally.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-echo "TODO: Read Netlify logs (when available via https://github.com/netlify/cli/issues/119 )"
\ No newline at end of file
+echo "TODO: Read Netlify logs (when available via https://github.com/netlify/cli/issues/119)"
```

Comment:
```suggestion
# intentional space before closing bracket so clickabel link is valid in CI output - otherwise it is also linked and link is invalid
echo "TODO: Read Netlify logs (when available via https://github.com/netlify/cli/issues/119 )"
```